### PR TITLE
FEATURE: add multiselect user custom field

### PR DESCRIPTION
**Pull request by lis2**
Ability to register new user custom field by plugins.

### CODE REVIEW

**Review by martin-brennan — APPROVED** [review]

**Comment by tgxworld** [comment]
@lis2 Is this intended for use in core? Otherwise, we should use the plugin API if we intend to allow plugins to extend this.

File: app/assets/javascripts/discourse/app/lib/plugin-api.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ import { replaceFormatter } from "discourse/lib/utilities";
 import { replaceTagRenderer } from "discourse/lib/render-tag";
 import { setNewCategoryDefaultColors } from "discourse/routes/new-category";
 import { addSearchResultsCallback } from "discourse/lib/search";
+import { addUserField } from "admin/models/user-field";
 
 // If you add any methods to the API ensure you bump up this number
 const PLUGIN_API_VERSION = "0.11.5";
```
Comment (tgxworld) [inline app/assets/javascripts/discourse/app/lib/plugin-api.js:79]: We have to bump the version here too.

**Review by tgxworld — APPROVED** [review]
@lis2 Looks good to me but do squash the commits with a proper commit title before merging :)

**Comment by tgxworld** [comment]
@lis2 This is probably a `FEATURE` instead of a `FIX`

**Review by tgxworld — APPROVED** [review]

**Pull request merged**